### ISSUE TITLE
Missing NO-46 subdivision ("Vestland") in Norway

### ISSUE THREAD

**Issue by ClaudioFloreani** [countries/countries#647]
Vestland is a county in Norway established on 1 January 2020.
Following values are from OpenStreetMap.

```
NO-46:
  unofficial_names: 
  translations: 
    en: 
  geo:
    latitude: 60.9291011
    longitude: 6.107886941976156
    min_latitude: 59.4754202
    min_longitude: 4.0875274
    max_latitude: 62.3823948
    max_longitude: 8.322053
  name: Vestland
```

**Comment by jufemaiz** [comment]
Pull request this change?

**Comment by pmor** [comment]
This has been fixed in #689
